x/staketia/keeper/msg_server.go

```diff
@@ -29,7 +29,7 @@ func (k msgServer) LiquidStake(goCtx context.Context, msg *types.MsgLiquidStake)
 // User transaction to redeem stake stTokens into native tokens
 func (k msgServer) RedeemStake(goCtx context.Context, msg *types.MsgRedeemStake) (*types.MsgRedeemStakeResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	nativeToken, err := k.Keeper.RedeemStake(ctx, msg.Redeemer, msg.StTokenAmount)
+	nativeToken, err := k.Keeper.RedeemStake(ctx, msg.Redeemer, msg.Receiver, msg.StTokenAmount)
```

Comment:
I don't understand the need for the receiver and how it connects to spillover. I also don't see us passing a Receiver into `NewMsgRedeemStake`.

Reply:
Great call! It was missing from `NewMsgRedeemStake`. Just added

And we need it cause stakeibc takes a receiver field but staketia did not originally

Reply:
lgtm. resolved.